### ISSUE TITLE
Consider using the vanilla CC0 waiver with a separate explanation of license status

### ISSUE THREAD

**Issue by stvnrlly** [18F/open-source-policy#55]
Because we modify the CC0 waiver, GitHub's license detector considers our repos to be unlicensed. We may be able to avoid this by using the unmodified CC0 waiver in `LICENSE.md` and a separate file that describes our public domain status as well as any third-party licenses that need explanation.

This may have the benefit of creating a standard place for putting third-party license information, as I think the practice of including it in `LICENSE.md` is not widespread.

**Comment by konklone** [comment]
I'd like @benbalter's take here, but offhand, I don't love splitting our relevant license info out into multiple files. Whether or not we've given appropriate attribution/notice/etc depends on how well we can expect people to learn our license information, and the `LICENSE` file pattern is relied heavily upon in GitHub for this.

**Comment by stvnrlly** [comment]
For me, the license file is not a great place to make attribution or notice. Others may not operate this way, but I usually assume that the file is the default version of the license specified in the `README` and wouldn't look there to find additional information.

**Comment by konklone** [comment]
That's a good point, and we do include the relevant information in our `README` template, but not all projects use that (and it's harder to automate the way we do our `LICENSE` and `CONTRIBUTING` files). If it wasn't detailed in the README, then it's split out and requires hunting. Also, requiring each of our repos to have two separate files with license information is more complicated than one.

Anyway, I am open to change. 😺 But I'm pointing out the downsides, and would like other input.

**Comment by benbalter** [comment]
> GitHub's license detector considers our repos to be unlicensed... I'd like @benbalter's take here

To clarify, it detects the repository as licensed, but under a non-standard license. This is the intended behavior, as the project is not licensed under the vanilla CC0 license.

Moving the "As a work of the United States Government, this project is in the public domain within the United States." line from the license file elsewhere, would cause it to be detected at CC0, but I'm not sure that's better than the current state, which reflects its unique licensing. 

If the US Federal Government would like to create its own standardized CC0/Public domain license (read: PLEASE DON'T!), we could detect that.

IAMYL, but I suspect the disclaimer could be removed entirely, and the project *actually* released under vanilla CC0, without brining 17 USC § 105 into the mix. IIRC, the public domain line (I wrote one of the early versions) was added to a GPL or MIT licensed project, and thus was necessary. In this case, you're saying "This project is public domain in the united states. It's also public domain outside of the united states". Logically, that could be simplified to "This project is public domain everywhere" (e.g., CC0), without losing any meaning. 

TL;DR: It's the intended behavior given what I believe to be the project maintainer's intent, but would suggest the edge case licensing is not necessary and only serves to complicate things (and suggest a proposal as such may be better received now that government lawyers are a bit more comfortable with open source and CC0).

**Comment by afeld** [comment]
Closing as stale.
